### ISSUE TITLE
feature scaling ,not able to execute

### ISSUE THREAD

**Issue by md-faiz** [md-faiz/hello-world#1]
not able to execute this code in python.

'X' got transformed but not 'y'

import numpy as np
import matplotlib.pyplot as plt
import pandas as pd
%matplotlib qt

# Importing the dataset
dataset = pd.read_csv('Position_Salaries.csv')
X = dataset.iloc[:,1:2 ].values
y = dataset.iloc[:, 2].values

"""# Splitting the dataset into the Training set and Test set
from sklearn.cross_validation import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.2, random_state = 0)"""

      #feature scaling
       
from sklearn.preprocessing import StandardScaler
sc_X = StandardScaler()
sc_y = StandardScaler()
X = sc_X.fit_transform(X)
y = sc_y.fit_transform(y.reshape(-1,1))

#fitting SVR model to the dataset
from sklearn.svm import SVR
regressor= SVR(kernel='rbf')
regressor.fit(X, y)
